compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/Clickable.kt

```diff
@@ -935,13 +929,14 @@ private class CombinedClickablePointerInputNode(
         interactionData.centreOffset = size.center.toOffset()
         detectTapGestures(
             onDoubleTap = if (enabled && onDoubleClick != null) {
-                { onDoubleClick?.invoke() }
+                { requestFocus(); onDoubleClick?.invoke() }
```

Comment:
Two expressions in 1 line to avoid merge conflicts?

Reply:
Yes, just a precaution, I less sure about 2 line change.